config/development.json

```diff
@@ -26,7 +26,7 @@
 	"dsp_widget_js_src": "https://widgets.wp.com/promote/widget.js",
 	"zendesk_presales_chat_key": "216bf91d-f10f-4f66-bf65-a0cba220cd38",
 	"features": {
-		"ad-tracking": false,
+		"ad-tracking": true,
```

Comment:
We shouldn't toggle this flag (it's fine for testing locally though), but now it will be enabled always in dev, which we don't want.